code/modules/client/preference_setup/occupation/occupation.dm

```diff
@@ -89,7 +89,7 @@
 
 		// In case we return 0 data from the database.
 		if (!jobs || !jobs.len)
-			pref.alternate_option	= 0
```

Comment:
The better place to do this would be in
`/datum/preferences/proc/new_setup(var/re_initialize = 0)`
(Which is the proc that is called if you click on "New Character" in the Character Load menu)

Reply:
It appears I missed that part. Updated it now, was that the correct place?